docs/user/reporting/reporting-troubleshooting.asciidoc

```diff
@@ -15,6 +16,11 @@ Having trouble? Here are solutions to common problems you might encounter while
 * <<reporting-troubleshooting-puppeteer-debug-logs>>
 * <<reporting-troubleshooting-system-requirements>>
 
+[float]
+[[reporting-troubleshooting-diagnostics]]
+=== Reporting Diagnostics
+Reporting comes with a built-in utility to try and find common issues automatically. When Kibana is running, navigate to the Report Listing page, and click the "Run reporting diagnostics..." button. This will open up a diagnostic tool that allows you to check various parts of reportings technology stack, and see where problems might be.
```

Comment:
```suggestion
Reporting comes with a built-in utility to try to automatically find common issues. When Kibana is running, navigate to the Report Listing page, and click the "Run reporting diagnostics..." button. This will open up a diagnostic tool that checks various parts of the Kibana deployment and produces a list recommendations that are based on the troubleshooting guide.
```

Reply:
Or maybe:
> This will open up a tool that tests various parts of the Reporting setup against common diagnostic problems.